src/repositories/media.repository.ts

```diff
@@ -678,10 +678,17 @@ export class MediaRepository extends Repository {
   }
 
   // tip: id = savedFeed.items()[0].media.id
-  public async save(mediaId: string) {
+  public async save(mediaId: string, collections_ids: string[] = []) {
```

Comment:
You can make this as an optional parameter (`collection_ids?: string[]`). It's more descriptive imo.